app/assets/stylesheets/application.css

```diff
@@ -141,9 +164,15 @@ fieldset.fieldset legend {
     margin-right: 0;
   }
 
+  .navigation ul li.active {
+    border-bottom: 0;
+    font-weight: 500;
```

Comment:
The underline looked odd on responsive, so I switched to bold for smaller screens:

<img width="1467" alt="Screen Shot 2024-05-08 at 2 15 55 PM" src="https://github.com/nodunayo/speakerline/assets/6457189/9b319885-b0bc-419e-9e18-3dd0c158cbd9">